src/server/config.rs

```diff
@@ -49,58 +52,111 @@ pub enum GrpcCompressionType {
 }
 
 /// Configuration for the `server` module.
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Configuration)]
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct Config {
     #[serde(skip)]
     pub cluster_id: u64,
```

Comment:
Missing `#[config(skip)]`

Reply:
Yeah, oversight on my part. Fixed it up now, though I still can't figure why the integrations are failing.

Reply:
Perhaps cause by some unstable tests...